### ISSUE TITLE
Text is insufficient contrast for accessibility

### ISSUE THREAD

**Issue by msecret** [18F/cg-dashboard#921]
The space (`dev` in screenshot) text supplies insufficient contrast for it's current size and color.

The color is `#8e8e8e` on a white (`#ffffff`) background, which is insufficient contrast for AA/AAA for small text. It does past the test for AA Large text, but large text is defined as ~18px + bold OR 24px, which this text is neither of.

![screen shot 2017-01-04 at 4 51 35 pm](https://cloud.githubusercontent.com/assets/1701077/21664643/bbb31376-d29c-11e6-882c-a9ec509e4fcb.png)

**Comment by msecret** [comment]
Another problematic text:

The green health color for entities is also of two low contrast. The color is `#0baf00` and fails for AA/AAA both small and large text.

![screen shot 2017-01-04 at 5 01 05 pm](https://cloud.githubusercontent.com/assets/1701077/21664858/0819349c-d29e-11e6-8d69-6b011099a01a.png)

**Comment by thisisdano** [comment]
I've cut the space path here: https://github.com/18F/cg-dashboard/pull/929
Green text I'd like to consider TBD. I don't think a simple color swap is the right move here.
